config/initializers/district_config.rb

```diff
@@ -0,0 +1 @@
+DistrictConfig.set_remote_filenames
```

Comment:
This seems like it should be more generic and describe why it's doing this, maybe `load!` or something.

Reply:
Good suggestion!